### ISSUE TITLE
IBM Equal Access Accessibility Checker identifies combobox as not having an associated label even if one is provided via input. 

### ISSUE THREAD

**Issue by ozatici** [IBM/carbon-components-angular#1789]
## Detailed description

> Describe in detail the issue you're having.

Though we are providing a label for combobox as input ( [label]="example" ), we are being flagged by the accessibility scan as not providing a label

<img width="787" alt="Screen Shot 2021-04-07 at 10 52 20" src="https://user-images.githubusercontent.com/18648691/113887353-6bd26480-978f-11eb-86f8-c84bfd539d00.png">
<img width="788" alt="Screen Shot 2021-04-07 at 10 52 14" src="https://user-images.githubusercontent.com/18648691/113887363-6c6afb00-978f-11eb-9777-71bf9ce6cf06.png">

> Is this a feature request (new component, new icon), a bug, or a general issue?
A bug

> Is this issue related to a specific component?
Combobox 

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

Carbon 10

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

**Comment by Donisius** [comment]
This may have already been addressed in #1641 which is available in version `4.29.2`. I'm gonna close this but please feel free to reopen it if the issue still persists
